src/gda/bnxt/backend_gda_bnxt.cpp

```diff
@@ -75,11 +75,8 @@ void GDABackend::bnxt_initialize_gpu_qp(QueuePair* gpu_qp, int conn_num) {
   gpu_qp->sq.mtu         = ibv_mtu_to_int(portinfo.active_mtu);
 
   /* Export DB */
-  err = bnxt_re_dv.get_default_db_region(context, &db_region_attr);
-  CHECK_ZERO(err, "bnxt_re_dv_init_obj(QP)");
-
-  CHECK_HIP(hipHostRegister(db_region_attr.dbr, getpagesize(), hipHostRegisterDefault));
-  CHECK_HIP(hipHostGetDevicePointer((void**) &gpu_qp->dbr, db_region_attr.dbr, 0));
+  CHECK_HIP(hipHostRegister(bnxt_qps[conn_num].db_region_attr->dbr, getpagesize(), hipHostRegisterDefault));
```

Comment:
Should this be `hipHostRegisterDefault`? Or is `hipExtHostRegisterUncached` or `hipHostRegisterIoMemory` more applicable? The latter seems to be new as of ROCm 7.1.1.